.github/CONTRIBUTING.md

```diff
@@ -108,7 +108,10 @@ Of course, before embarking on such a journey, [discuss with the core team](http
 
 ## Merging Pull Requests (PRs)
 
-The preferred order of [merging PRs](https://help.github.com/articles/about-pull-request-merges/) is: prefer rebase, else squash if you'd like to clean up the history before merging. Avoid doing a merge commit.
+The preferred order of [merging PRs](https://help.github.com/articles/about-pull-request-merges/) is: 
+1. Prefer _squash and merge_ for a clean history and added PR numbers for details of discussion for future comparison. 
+2. Else _rebase and merge_ if you'd like to keep the history, but know this will not link to the PR in changelog, etc.
```

Comment:
I think we should clarify which changelog - that is the PTB one

Reply:
Not sure why this wouldn't affect regular releases?

Reply:
Regular releases get the changelog messages get rewritten so they make (some) sense for the users

Reply:
Do we want to point out that it is wise to review and edit the individual commits' message(s) that get concatenated and presented after the "Squash and merge" button is clicked but before the "Confirm squash and merge" one is?

This is probably desirable to remove the (almost) duplicate first line showing in the editor which is the header line for the first commit (prefixed with a `*`) and which will like have been copied but amended to have that PR number at the end. Otherwise one ends up with a commit message like that of c32bac686c5132c280803fed850b1b6aaf741340 :
>  Fix macOS Actions builds (#4327)
>
>\* Fix macOS Actions builds
>
>\* Rearchitect to be simpler
>
>\* Try building without a prefix path

For a *single-commit* PR that duplicate line is (to *some* OCD people) *irritatingly* pointless...

Reply:
Put your improvement proposals on https://github.com/Mudlet/Mudlet/issues/new so they do not get lost in scroll here